packages/backend/src/tokens/request.ts

```diff
@@ -66,6 +66,10 @@ export async function authenticateRequest(
   const authenticateContext = createAuthenticateContext(createClerkRequest(request), options);
   assertValidSecretKey(authenticateContext.secretKey);
 
+  const cookiePipeline = new Pipeline<string, string>()
+    .addOperation(new SuffixCookieOperation(authenticateContext.publishableKey))
+    .addOperation(new UnSuffixCookieOperation(authenticateContext.publishableKey));
+
```

Comment:
Why do we define the pipeline here instead of closer to where we call it? Is it used by something in between?